### DISCUSSION TITLE
24.03.13(수) 회의

### DISCUSSION THREAD

**Discussion by Shinjiwoooo** [NC4-FinalProject/NC4_Final_Project_2.14#7] [Announcements]
# 회의 내용

## Github 
- 깃허브 Discussions에 회의록 및 공지사항 기록예정
- 깃허브  Milestone(대분류)안에  Issues(세부컴포넌트) 기능 작업 단위 기록 및 task done단위, lables 등록, project 작업 진척도 적극 활용

## Branch 전략
- **main**
- **hotfix**
- **dev**
    - 240313
    - 240314
- **developer**
    - jw
    - jk
    - jb
    - ms
    - hj
    - th
![image](https://github.com/NC4-FinalProject/NC4_Final_Project_2.14/assets/148425731/1d215204-b4fa-4b3b-90cd-6f0ee22d902c)

### 미수
- 공통 속성
- Navi (MUI)
- Button
- SVG Button
- Hover Description

### 지우
- SearchInput
- TextInput
- Modal (가운데 표출)
- Tag

### 태현
-  Header
-  Footer
-  SelectBox
-  ToggleMenu (알림창 표출)

## Notion
+ 컨벤션 참고 https://www.notion.so/038f0f37c61e49fab23775a062f2d7ed
+ 컴포넌트 명세서 와 API명세서 작성 https://drive.google.com/drive/folders/1wpulH2l-fqbVc54gC-x8Kw3g4DQuewfT
(컴포넌트 생성 및 api 생성시 바로 작성)

## 참고 사이트 
+  PX  to REM 변환기 사이트 https://nekocalc.com/px-to-rem-converter
(기본 px 단위 꼭 정하고 변환하기)

## Git 올릴때 주의사항
- 매일 정해진 시간에 dev branch(날짜브랜치) 에 merge 후 최종적으로 main에 merge 시킨 후 매일 날짜로 새로운 브랜치 생성예정
- Commit, Push 할때 전체 팀원에게 말하고 올리기

## 다음 회의 주제
-  각자 맡은 기능 앞단 컴포넌트 개발  및 merge 작업

**Comment by HMisu** [comment]
확인 완료했습니다.

**Comment by Selah-on-Github** [comment]
내용 확인했습니다.

**Comment by KimJongBeomm** [comment]
확인 완료

**Comment by Robbie-Kim** [comment]
확인 완료입니다
